CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Removed requestAnimationFrame polyfill.
 - Removed `_tests/browser_tests/README.md`, `_tests/macro_testing/README.md`, `_tests/processor_tests/README.md`.
 - Removed `grunt vendor` from `setup.sh`.
+- Removed unused CSS
```

Comment:
No worries with updating this, but a quick note on the area where the CSS was removed may make the changelog more useful as a reference down the road. (see last entry in 0.2.0. https://github.com/cfpb/cfgov-refresh/blob/flapjack/CHANGELOG.md#fixed-10 :stuck_out_tongue_closed_eyes: )